css/bs-modal.css

```diff
@@ -0,0 +1,183 @@
+/*!
```

Comment:
`normalize.css` version 3.02 is already in the repos under `css/normalize.css`. We'll work together to see what we can do to merge the new dependencies in.